addons/cssresources/src/css-resource-panel.tsx

```diff
@@ -96,7 +98,24 @@ export class CssResourcePanel extends Component<Props, State> {
                 <input type="checkbox" checked={picked} onChange={this.onChange} id={id} />
                 <span>#{id}</span>
               </label>
-              {code ? <SyntaxHighlighter language="html">{code}</SyntaxHighlighter> : null}
+              {code && code.length < maxLimitToUseSyntaxHighlighter && (
+                <SyntaxHighlighter language="html">{code}</SyntaxHighlighter>
+              )}
+              {code && code.length >= maxLimitToUseSyntaxHighlighter && (
+                <div>
+                  <pre>{code.substring(0, maxLimitToUseSyntaxHighlighter)} ...</pre>
+                  <div
```

Comment:
Thank you @tomilaurell 

This should probably use a Placeholder component:
https://storybookjs.now.sh/official-storybook/?path=/story/basics-placeholder--two-children

This way it will style along nicely with the rest of storybook, and be possible for users to theme it.

There are also Icons you can use:
https://storybookjs.now.sh/official-storybook/?path=/story/basics-icon--labels

And we have this component for spacing:
https://storybookjs.now.sh/official-storybook/?path=/story/basics-spaced--row

Reply:
You can import all these components from `@storybook/components`

Reply:
I will make those changes tomorrow.

Reply:
Awesome, thanks!